thesis-gui/src/services/quizzes-service.ts

```diff
@@ -8,5 +8,17 @@ export const quizzesService = {
 
     createNew(questionsSet: QuestionsSetCreateRequest) {
         return httpService.post<QuestionsSetCreateRequest, QuestionsSetKeyResponse>('/questions-sets', questionsSet);
+    },
+
+    getQuiz(key: string) {
+        return httpService.get<QuizFullResponse>(`/questions-sets/${key}`);
+    },
+
+    update(key: string, questionsSet: QuestionsSetCreateRequest) {
+        return httpService.put<QuestionsSetCreateRequest, QuestionsSetKeyResponse>(`/questions-sets/${key}`, questionsSet);
+    },
+
+    delete(key: string) {
+        return httpService.delete<undefined, QuestionsSetKeyResponse>(`/questions-sets/${key}`, undefined);
```

Comment:
Why response type here is `QuestionsSetKeyResponse` while the return type of `deleteQuestionsSet` method in `TeacherQuestionsSetController` class is void? Shouldn't it be `undefined`?

Reply:
You're right. Changed to `undefined`.